cns/NetworkContainerContract.go

```diff
@@ -78,10 +78,12 @@ type NICType string
 // NIC Types
 const (
 	InfraNIC NICType = "InfraNIC"
-	// Delegated VM NICs are projected from VM to container network namespace
-	DelegatedVMNIC NICType = "DelegatedVMNIC"
```

Comment:
needs one change in dnc as well since this gets assigned there - https://msazure.visualstudio.com/DefaultCollection/One/_git/Networking-Aquarius?path=/src/dnc/datamodel/networkmanager.go&version=GBmaster&line=2864&lineEnd=2868&lineStartColumn=1&lineEndColumn=3&lineStyle=plain&_a=contents

Reply:
I think DNC will need to wait until this PR merged and **the new CNS release was out.** Otherwise DNC can't compile.

Reply:
Update: This change will not be applied to DNC currently due to version compatibility issue. Pls resolve